### ISSUE TITLE
Can it be used in images?

### ISSUE THREAD

**Issue by Fran-BR** [kmaninis/OSVOS-PyTorch#40]
The accuracy of the model looks very good. Can it be adapted for images?

**Comment by kmaninis** [comment]
Hello. OSVOS needs the annotated first frame of a video for fine-tuning. So probably it won't work in your use case, unless the images (and the object to be segmented) are very similar in appearance.
